src/Nest/XPack/RollUp/RollupSearch/ElasticClient-RollupSearch.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Threading.Tasks;
+using Elasticsearch.Net;
+using System.Threading;
+
+namespace Nest
+{
+	public partial interface IElasticClient
+	{
+		/// <inheritdoc/>
```

Comment:
I think this should have a `<summary>...</summary>` and all other overloads inheritdoc from it